src/modules/beforeScreenshot.js

```diff
@@ -34,7 +34,7 @@ export default async function beforeScreenshot(browser, options) {
   await browser.execute(scroll, x, y);
 
   // wait a bit for browser render
-  const pause = 200;
+  const pause = 5000;
```

Comment:
Why the bump to 5000? This is cause this to become tremendously slow if you are taking more than a few screenshots. Why not ensure the browser is rendered before attempting to take a screenshot in the test(s)?